examples/bluegreen/bluegreen-controller.yaml

```diff
@@ -1,19 +1,25 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: bluegreendeployments.ctl.enisoc.com
 spec:
   group: ctl.enisoc.com
-  version: v1
   scope: Namespaced
   names:
     plural: bluegreendeployments
     singular: bluegreendeployment
     kind: BlueGreenDeployment
     shortNames:
     - bgd
-  subresources:
-    status: {}
+  versions:
+  - name: v1
+    served: true
```

Comment:
is `served`/`storage` and  `schema` required by CRD version `v1`?

Reply:
Yes, `schema` is always required. And at least one version must he `served` and `stored`. Since we have only one version in this CRD, the fields should be specified.